app/controllers/api/users_controller.rb

```diff
@@ -26,5 +26,21 @@ def login
         }
       }.to_json
     end
+
+    def user_name
```

Comment:
Having an endpoint only retrieving the user's name is not that reusable. We already query the database for the entire user object, so let's change this endpoint to return also other details about the user, like the id and email.